src/use-swr.ts

```diff
@@ -275,7 +275,7 @@ export const useSWRHandler = <Data = any, Error = any>(
           // Keep the stale data but update error.
           setState({
             isValidating: false,
-            error: err
+            error: err as Error
```

Comment:
type guard for error

![image](https://user-images.githubusercontent.com/4800338/132836101-03faa679-a6b5-46cd-88ea-5e650e53dc0b.png)

not sure why I'm having this...since we're still not ts v4. Can also be disabled by `"useUnknownInCatchVariables": false` when upgrading to v4